developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md

```diff
@@ -0,0 +1,312 @@
+[Aptos status page](https://status.devnet.aptos.dev/). If your node is catching up to the current version, it is synchronizing correctly.
+
+:::note
+
+It is fine if the status page differs by a few versions, as the status
+page does not automatically refresh.
+
+:::
+
+### (Optional) Verify outbound network connections
+
+Optionally, you can check the output network connections. The number of outbound network connections should be more than `0`. Run the following command:
+
+```
+curl 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_connections{direction=\"outbound\""
+```
+
+The above command will output the number of outbound network connections for your node. For example:
+
+```
+$ curl 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_connections{direction=\"outbound\""
+aptos_connections{direction="outbound",network_id="Public",peer_id="aabd651f",role_type="full_node"} 3
+```
+
+If the number of outbound connections returned is `0`, then it means your node cannot connect to the Aptos blockchain. If this happens to you, follow these steps to resolve the issue:
+
+1. Update your node to the latest release by following the [update instructions](#update-fullnode-with-new-releases).
+2. Remove any `seed` peers you may have added to your `public_full_node.yaml` configuration file. The seeds may be preventing you from connecting to the network. Seed peers are discussed in the [Add upstream seed peers](#add-upstream-seed-peers) section.
+
+### (Optional) Examine Docker ledger size
+
+The blockchain ledger's volume for Aptos devnet can be monitored by entering the Docker container ID and checking the size.
+This will allow you to see how much storage the blockchain ledger is currently consuming.
+
+- First, run `docker container ls` on your terminal and copy the NAME field output. This will be a string similar to `public_full_node_fullnode_1`.
+- Next, run these commands to check the storage size consumed by the ledger, using the NAME field you copied over in place of `public_full_node_fullnode_1`:
+
+```
+# Obtain the container ID:
+id=$(docker container ls | grep public_full_node_fullnode_1 | grep -oE "^[0-9a-zA-Z]+")
+# Enter the container:
+docker exec -it $id /bin/bash
+# Observe the volume (ledger) size:
+du -cs -BM /opt/aptos/data
+```
+
+## Add upstream seed peers
+
+:::tip
+
+You may see `NoAvailablePeers` in your node's error messages. This is normal when the node is first starting.
+Wait for the node to run for a few minutes to see if it connects to peers. If not, follow the below steps:
+
+:::
+
+Devnet validator fullnodes will only accept a maximum of ~15000 connections. If Aptos devnet is experiencing high network traffic volume, your FullNode might not able to connect and you may see `NoAvailablePeers` continuously in your node's error messages. If this happens, manually add peer addresses in the `seeds` key in `public_full_node.yaml`, the FullNode configuration file. This will then connect your FullNode to the specified seed peer.
```

Comment:
I'm not sure we should have this number since it may change from time to time.

Also, technically it's not high network traffic, but high connection volume (large number of full nodes connected).